### ISSUE TITLE
Decide on the target we want to have for accessibility and lighthouse checks

### ISSUE THREAD

**Issue by choldgraf** [jupyter/jupyter.github.io#793]
In https://github.com/jupyter/jupyter.github.io/pull/792 we lowered the threshold for our lighthouse checks in order to un-break our CI/CD and avoid the "expected failures" anti-pattern. This is a follow-up issue to recognize the fact that we want to set targets for accessibility and jobs like lighthouse that balance _realistic_ and also _solve the most important accessibility problems_. 

The crux of the matter is that we have very limited resources, so we need to choose targets for accessibility that are realistic given these resources. We also need to decide that lighthouse scores are a "good" reflection of our targets (maybe based on best practices for how a11y experts recommend using lighthouse).
